x-pack/platform/plugins/shared/cases/server/client/cases/get.ts

```diff
@@ -281,6 +288,7 @@ export const resolve = async (
         savedObject: resolvedSavedObject,
         comments: theComments.saved_objects,
         totalComment: theComments.total,
+        totalEvents: countEventsForID({ comments: theComments }),
```

Comment:
I find you can attach an event to a case multiple times, which causes a mismatch between the badge counts and table

<img width="990" height="536" alt="image" src="https://github.com/user-attachments/assets/1ed73625-50d3-4e51-96fc-f46079667693" />

Reply:
hey, while it is still possible to call the attachment action on the same event twice, it is now filtered on the backend side (same as the alerts) - so this should be fixed. thank you!